artist/forms.py

```diff
@@ -25,6 +25,10 @@ class ArtistApplyForm(forms.Form):
             attrs={"placeholder": "We started playing music because..."}
         )
     )
+    project = forms.CharField(
+        label="Project Name",
+        widget=forms.TextInput(attrs={"placeholder": "Single/Album Name"}),
+    )
```

Comment:
The addition of this requires an update to the `testArtistApplication()` test. Check out `artist/tests.py:ArtistWebTestCase.testArtistApplication()`.